### ISSUE TITLE
Add option to filter where `@edgedb/generate queries` looks for `.edgeql` files.

### ISSUE THREAD

**Issue by jaclarke** [edgedb/edgedb-js#617]
Currently the `npx @edgedb/generate queries` command will generate queries for every `.edgeql` in your edgedb project (defined by the edgedb.toml), but you may not want some `.edgeql` files to be included, eg. bootstrap scripts, or have multiple projects sharing the same edgedb project, eg. monorepos.

So it would be useful to be able to configure where it looks for `.edgeql` files, to exclude certain directories, and/or only include certain directories, possibly using glob expressions?.

**Comment by scotttrinh** [comment]
Yeah, that seems reasonable. I can imagine something like what prettier and eslint use as their path resolution should work for us.
